src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -263,7 +263,12 @@ def run_adcm(
         else:
             container_ip = ip
 
-        wait_for_url("http://{}:{}/api/v1/".format(container_ip, port), 60)
+        init_timeout = 120
+        if not wait_for_url(
+            "http://{}:{}/api/v1/".format(container_ip, port), init_timeout
+        ):
+            container.stop()
```

Comment:
Use [kill](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.kill) instead of stop. Stop may be time consuming.

Reply:
Done